src/api/itemApi.tsx

```diff
@@ -0,0 +1,62 @@
+import axios from 'axios';
+
+const BASE_URL = 'https://panda-market-api.vercel.app';
+
+export async function getProducts(params = {}) {
+    const allowedParams = ["orderBy", "pageSize", "page", "keyword"];
+    const invalidParams = Object.keys(params)
+        .filter(key => !allowedParams.includes(key));
+        
+    if (invalidParams.length > 0) {
+        console.error(`Invalid parameters detected: ${invalidParams.join(", ")}`);
+    }
+
+    try {
+        const response = await axios.get(`${BASE_URL}/products`, {
```

Comment:
P1:
가능하면 어떤 값이 return될지 아래처럼 작성해주세요~

```suggestion
        const response = await axios.get<ProductProps>(`${BASE_URL}/products`, {
```